include/eve/module/arithmetic/regular/ieee.hpp

```diff
@@ -0,0 +1,51 @@
+//==================================================================================================
```

Comment:
This file should not exist. 
If IEEE is in arithmetic, those thing should be in arithmetic.